foxy/distribution.yaml

```diff
@@ -5713,23 +5713,6 @@ repositories:
       url: https://github.com/ros2/rosidl_typesupport_fastrtps.git
       version: foxy
     status: developed
-  rosidl_typesupport_gurumdds:
-    doc:
-      type: git
-      url: https://github.com/ros2/rosidl_typesupport_gurumdds.git
-      version: foxy
-    release:
-      packages:
-      - gurumdds_cmake_module
-      tags:
-        release: release/foxy/{package}/{version}
-      url: https://github.com/ros2-gbp/rosidl_typesupport_gurumdds-release.git
-      version: 1.1.1-1
-    source:
-      type: git
-      url: https://github.com/ros2/rosidl_typesupport_gurumdds.git
-      version: foxy
-    status: developed
```

Comment:
Depending on the iteration process here, if we are otherwise ready to merge this while Foxy is still in a sync hold I'll split the changes for foxy out into a separate pull request before merging.